docs/Makefile

```diff
@@ -17,5 +17,7 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	-@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-	git clean -dfx source/
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	git clean -dfx source/examples
+	git clean -dfx source/tutorials
+	git clean -f source/getting_started.rst
```

Comment:
be more explicit about what we want to clean. Otherwise, this will clean any new file in `source` folder